src/externalApi.ts

```diff
@@ -2,11 +2,11 @@ import axios from "axios";
 
 export const addImageFlag = ({ barcode, imgid, url }) => {
   axios
-    .put(`https://amathjourney.com/api/off-annotation/flag-image/${barcode}`, {
-      mode: "no-cors",
-      imgid,
-      url,
-    })
+    .put(
+      `https://amathjourney.com/api/off-annotation/flag-image/${barcode}`,
```

Comment:
I forgot it. This was a nintial experiment. But my personall server is down since a year or two 🙃

It should be replaced by a call to nutripatrol

https://nutripatrol.openfoodfacts.org/api/docs#/default/create_flag_api_v1_flags_post